.vitepress/config.ts

```diff
@@ -76,9 +77,10 @@ export default defineConfig(({ mode }) => {
 							{ text: 'IntelliJ', link: '/admin/Entwicklungsumgebungen/IntelliJ/' },
 							{ text: 'macOS', link: '/admin/Entwicklungsumgebungen/macOS/' },
 							{ text: 'VS-Code', link: '/admin/Entwicklungsumgebungen/VS-Code/' },
-							{ text: 'Code-Vorgaben', link: '/admin/Entwicklungsumgebungen/Coding-Guidlines'},
-							{ text: 'Code-Styles', link: '/admin/Entwicklungsumgebungen/Code-Styles'},
-						] },
+							{ text: 'Coding Guidlines', link: '/admin/Entwicklungsumgebungen/Coding-Guidlines' },
+							{ text: 'Code Styles', link: '/admin/Entwicklungsumgebungen/Code-Styles' }
```

Comment:
bitte immer ein Komma hinter das letzte element in solchen mehrzeiligen Listen. Wenn man ein weiteres Element ergänzt, dann wird die zuvorgehende Zeile wegen dem Komma nicht zusätzlich im Diff markiert. Erleichtert das Lesen zukünftiger diffs